.buildkite/comparison/pipeline.sh

```diff
@@ -13,7 +13,7 @@ profiling=disable
 # set up environment and agents
 cat << EOM
 env:
```

Comment:
This looks broken and will not run. For example, the module cuda/11.3 doesn't exist. We should remove this pipeline

Reply:
Ok, we can fix the rest of what is broken later, for now, I'm just trying to keep versions in sync.